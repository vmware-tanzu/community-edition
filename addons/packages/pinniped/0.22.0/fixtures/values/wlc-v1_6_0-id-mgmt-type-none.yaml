#@data/values
---
infrastructure_provider: vsphere
tkg_cluster_role: workload
identity_management_type: none
pinniped:
  cert_duration: 2160h
  cert_renew_before: 360h
  supervisor_svc_endpoint: https://10.191.180.176
  supervisor_ca_bundle_data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURNRENDQWhpZ0F3SUJBZ0lRWGhEdnoxcXRzQU1aV2VtclhlcjZ2akFOQmdrcWhraUc5dzBCQVFzRkFEQVgKTVJVd0V3WURWUVFERXd4MGEyY3RjR2x1Ym1sd1pXUXdIaGNOTWpJd05URXlNVGd6TVRVeldoY05Nekl3TlRBNQpNVGd6TVRVeldqQVhNUlV3RXdZRFZRUURFd3gwYTJjdGNHbHVibWx3WldRd2dnRWlNQTBHQ1NxR1NJYjNEUUVCCkFRVUFBNElCRHdBd2dnRUtBb0lCQVFDY29GL3lXbVRUKzdNQmJMbWx3UWZnVzZ6ZU92Z1BaS3J3RjQ5aVNDTnkKQ1BTditIM29Jb0xsSE8xWnZYbXpQU3dUeVR1LzNQOXEyRUIxdUh5OXhoV2xQNWExbTVKRHFPeU1SNmN2M25hTwptbVNiT3ZRdGZLY2g0dUtVMk1vSkxlWGtDNVNiQW95eVRKQk1DS1ZydkhpcWUrTUgzcTFDaWVic0pha3g4cEdkCmNwbERsUTQvekJNRTduWS9MSzUzZDdncnNzZGRxdDVKMUVScmdINUNHOEVVSi8xMi9DdklMSGllWG40RXJVcmgKQmVBaUtFVCsvcG1kOFJTcjBMYU1DN0lQWFlCNGY4VzlVS1UvVDVMNm1Ob2h5amNzdnNRTDR3bjh6WDh5WVp1Wgp3WFVLWms5TFFnVmowWXAvdm5QY3JoaUg3RVM4NytoakxoeEJPQmlQVmM5aEFnTUJBQUdqZURCMk1BNEdBMVVkCkR3RUIvd1FFQXdJQ0JEQWRCZ05WSFNVRUZqQVVCZ2dyQmdFRkJRY0RBUVlJS3dZQkJRVUhBd0l3RHdZRFZSMFQKQVFIL0JBVXdBd0VCL3pBZEJnTlZIUTRFRmdRVS8vL3QvZFBrT1BKZmtnc0ZaaTZHOG5VTFJUTXdGUVlEVlIwUgpCQTR3RElJS2NHbHVibWx3WldSallUQU5CZ2txaGtpRzl3MEJBUXNGQUFPQ0FRRUFQV2V1WnorVHNhbEdxS3daClVwUTR2dmQ0NWhBUnFHcmdIQXlEU2g1N0swSU13R2Y3MEVESHFiWFEwK1RCVW5PNTdXb2JNVWFhZnMzSldieEsKc1JjbVZNb283TnJqTnZ1emw3NG5zMnI4V2F4M1FpcEhpelZQRWIrSEhOY2FuQU9WMXZQV0RaeUVFZFVUZGhQdgpuaHBJbHA1alJWaWVjQ0xRYWRDNjU0RlFkQk9HVkxmYVQvQjZFRytNbEt0dy9iNTRJQ0RLSE5FNStJSllmQXFXCnN2eW5FM3lZaUlTTDJXOCt3anY3UkpCbk54M0QwRTNoUWtNOTBTNHo4Y0phaDhsUG16dkFaK3J4L3dEajByeWsKdDZVUEkyOHVMYWNiNVVZQUM4ZldZZmtzMW94ZWdOcUp5cHhmTnlPUW5JTFBSM2xCOTNCWnlFSGhJeEtmbjg2ZApsSWhkOHc9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
  image:
    name: pinniped
    tag: v0.12.1_vmware.1
    repository: projects-stg.registry.vmware.com/tkg
    pull_policy: IfNotPresent
  post_deploy_job_image:
    name: tanzu_core/addons/tkg-pinniped-post-deploy
    tag: v0.22.0-dev-23-ga3552d03
    repository: projects-stg.registry.vmware.com/tkg
    pull_policy: IfNotPresent
