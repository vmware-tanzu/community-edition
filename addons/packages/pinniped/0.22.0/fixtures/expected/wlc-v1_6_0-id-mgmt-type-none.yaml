#! Yes, this file is empty.
#!
#! It is because when we set identity_management_type: none, we expect no resources to get generated.
